static/app/views/settings/project/projectFilters/projectFiltersSettings.tsx

```diff
@@ -537,9 +538,22 @@ export function ProjectFiltersSettings({project, params, features}: Props) {
                       type: 'boolean',
                       name: 'filters:react-hydration-errors',
                       label: t('Filter out hydration errors'),
-                      help: t(
-                        'React falls back to do a full re-render on a page and these errors are often not actionable.'
-                      ),
+                      help: organization.features.includes(
```

Comment:
if they're not under the feature flag, should we add a comment explaining that they don't have access or that sort of thing?

Reply:
i think if they're not under the feature flag they don't need to know about the replay-hydration settings page. we're just testing it basically, so if it all went away it could be confusing to folks. they'll get it when it's released!

If they don't have replay at all though.. then that's a good question. We could link to the page and have the whole page be disabled or something like that.